lib/napa_rabbit_publisher/rabbit_active_record_callbacks.rb

```diff
@@ -13,6 +14,10 @@ def post_update_to_rabbit
       post_to_rabbit('updated')
     end
 
+    def post_destroy_to_rabbit
+      post_to_rabbit('destroy')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.